documentation/Logical Volume Manager (LVM) format.asciidoc

```diff
@@ -291,8 +293,7 @@ Contains a list of strings. See section: <<status_flags,Status flags>>
 | flags | The flags +
 Contains a list of strings. See section: <<flags,Flags>>
 | extent_size | The size of an extent +
-The value contains the number of sectors +
-[yellow-background]*Note that the sector size is currently assumed to be 512 bytes.*
+The value contains the number of sectors
```

Comment:
please do not remove notes regarding assumptions (repeat this across your changes)